css/gora.css

```diff
@@ -0,0 +1,540 @@
+body {
```

Comment:
open your project with devtools opened and check if css property changes UI or not and remove all unnecessary !!